### PR TITLE
Add java.util.Arrays patch to use gpu module code

### PR DESCRIPTION
**Pull request by theresa-m**
Fixes: https://github.com/eclipse/openj9/issues/189

Dependent PR: https://github.com/eclipse/openj9/pull/1485

Signed-off-by: Theresa Mammarella <Theresa.T.Mammarella@ibm.com>

### CODE REVIEW

**Comment by theresa-m** [comment]
Can someone please advise on what is incorrect with the Arrays.java copyright?

**Comment by DanHeidinga** [comment]
> FAILURE - Copyright date in file: 'jdk/src/java.base/share/classes/java/util/Arrays.java' appears to be incorrect

My guess would be a missing comma between 2018 2018

**Comment by DanHeidinga** [comment]
```
+/*
 + * Licensed Materials - Property of IBM
 + * RMI-IIOP v1.0
 + * Copyright IBM Corp. 2018 2018  All Rights Reserved
 + *
 + */
```
This doesn't match the copyrights in other files in this repo.  Are you sure this is correct?

**Comment by theresa-m** [comment]
I found some java files with that copyright - mostly in the corba directory. Found this one as well:
```
/*
 * ===========================================================================
 *
 * (C) Copyright IBM Corp. 2003 All Rights Reserved.
 *
 * ===========================================================================
 */
```

**Comment by DanHeidinga** [comment]
Corba may be a special case due to the development of the initial implementation.

**Comment by theresa-m** [comment]
got it. thanks @DanHeidinga !

File: jdk/src/java.base/share/classes/java/util/Arrays.java [hunk 1]
```diff
@@ -83,6 +90,26 @@
     // Suppresses default constructor, ensuring non-instantiability.
     private Arrays() {}
 
+    private static final class GPUAssistHolder {
```
Comment (keithc-ca) [inline jdk/src/java.base/share/classes/java/util/Arrays.java]: Please promote this to a package-private class to reduce the potential for merge conflicts.

**Comment by theresa-m** [comment]
^ Moved GPUAssistHolder.java to separate java file in eclipse/openj9 repo (see eclipse/openj9#1485)

**Review by keithc-ca — APPROVED** [review]

**Pull request merged**